__tests__/app.test.js

```diff
@@ -92,4 +92,48 @@ describe('app.js', () => {
             })
         })
     })
+
+    describe('GET /api/articles', () => {
+        test('200: responds with all articles', () => {
+            return request(app).get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+                const {articles} = body
+                expect(articles.length > 0).toBe(true)
+                articles.forEach((article) => {
+                    expect(article.hasOwnProperty('author')).toBe(true)
+                    expect(typeof article.author).toBe('string')
+
+                    expect(article.hasOwnProperty('title')).toBe(true)
+                    expect(typeof article.title).toBe('string')
+
+                    expect(article.hasOwnProperty('article_id')).toBe(true)
+                    expect(typeof article.article_id).toBe('number')
+
+                    expect(article.hasOwnProperty('topic')).toBe(true)
+                    expect(typeof article.topic).toBe('string')
+
+                    expect(article.hasOwnProperty('created_at')).toBe(true)
+                    expect(typeof article.created_at).toBe('string')
+
+                    expect(article.hasOwnProperty('votes')).toBe(true)
+                    expect(typeof article.votes).toBe('number')
+
+                    expect(article.hasOwnProperty('article_img_url')).toBe(true)
+                    expect(typeof article.article_img_url).toBe('string')
+
+                    expect(article.hasOwnProperty('comment_count')).toBe(true)
+                    expect(article.hasOwnProperty('body')).toBe(false)
+                })
+            })
+        })
+        test('200: articles are sorted by date in descending order', () => {
+            return request(app).get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+                const {articles} = body
+                expect(articles).toBeSorted({ key: 'created_at', descending: true})
```

Comment:
Very nice! If you want you can remove line 132 `.expect(200)` because that has been covered in the test before this one